wgengine/magicsock/magicsock.go

```diff
@@ -1318,7 +1318,14 @@ func (c *Conn) sendDiscoMessage(dst netip.AddrPort, dstKey key.NodePublic, dstDi
 		// Can't send. (e.g. no IPv6 locally)
 	} else {
 		if !c.networkDown() {
-			c.logf("magicsock: disco: failed to send %v to %v: %v", disco.MessageSummary(m), dst, err)
+			if isPMTUError(err) {
```

Comment:
This assumes `m disco.Message` is a non-default sized disco Ping prepared by PMTU logic.

Reply:
if `m` is a `Disco.Ping` with a nonzero `Padding` and `err` is EMSGSIZE and `debugPMTUD()` is false, then don't log, otherwise log?

Alternatively callers of sendDiscoMessage() could signal behavior via an arg